lib/pipeline/pipeline.js

```diff
@@ -141,11 +175,25 @@ class Pipeline {
                 }).join("\n");
 
                 self.logger.info(__("writing file") + " " + (self.buildDir + targetFile).bold.dim);
+                if(new RegExp(/^index.html?/i).test(targetFile)){
```

Comment:
What happens if thee is two or more index.html files? Could happen if someone has multiple folders.

Reply:
Good look @jrainville! I tested this by creating a folder `app/subfolder/index.html` and added that file to `embark.json`. This works as is because the `^` enforces start of the string. Any other index.html files will live in subfolders and will have the subfolder name at the start of the string.